.github/scripts/main/test.sh

```diff
@@ -26,3 +26,9 @@ eval $(opam env)
 opam install lwt
 opam list
 opam config report
+
+# Make sure opam init (including the the sandbox script) works in the absence of OPAMROOT
+unset OPAMROOT
+rm -r ~/.opam
+opam init --bare
```

Comment:
`-vvv` to see sandbox check ? I don't know if we want to, maybe to debug in case of failure ?